GiniVision/Classes/GiniConfiguration.swift

```diff
@@ -188,6 +188,15 @@ import UIKit
      */
     public var qrCodeScanningEnabled = false
     
+    /**
+     Indicates the status bar style on the Gini Vision Library.
+     
+     - note: If `UIViewControllerBasedStatusBarAppearance` is set to `false` in the `Info.plist`,
+     it may not work in further versions of iOS since `UIApplication.setStatusBarStyle` method was
```

Comment:
since **the** `UIApplication.setStatusBarStyle` method.

I would also consider changing "further" to "future".